### DISCUSSION TITLE
How to shrink the space between homeInfo and article directory？

### DISCUSSION THREAD

**Discussion by JhuoW** [adityatelange/hugo-PaperMod#872] [Q&A]
I want to reduce the space between HomeInfo and head/articles.  As shown below, I want to reduce the height of the red box. How to set it?
![papermod](https://user-images.githubusercontent.com/12757730/161598162-0f967bb5-33c6-4254-809a-deb5d98be6f2.png)

**Comment by ngntrgduc** [comment]
I had the same question and finally, I figured it out.

Create a file `assets\css\common\post-entry.css` in your root folder, and then copy the same file of the theme (`themes\PaperMod\assets\css\common\post-entry.css`) to it.

Then, at the `.first-entry` class, remove the `min-height: 320px;` or you can change the height you want.
